pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -300,7 +300,8 @@ pub mod pallet {
 	where
 		<CurveParameterTypeOf<T> as Fixed>::Bits:
 			Copy + ToFixed + AddAssign + BitOrAssign + ShlAssign + TryFrom<U256> + TryInto<U256>,
-		CollateralCurrenciesBalanceOf<T>: Into<U256> + TryFrom<U256> + TryInto<U256>,
+		CollateralCurrenciesBalanceOf<T>: Into<U256> + TryFrom<U256>,
+		FungiblesBalanceOf<T>: Into<U256> + TryFrom<U256>,
```

Comment:
I think this has also to be specified in the benchmarking

Reply:
5b3a4ef3c3ee30c74d4e9f58521e3f29af0c0843